doc/src/explanation/active-deprecations.md

```diff
@@ -1182,6 +1182,12 @@ See also {ref}`deprecated-poly-nonpoly-binary-operations` above.
 (deprecated-sympify-string-fallback)=
 ### The string fallback in `sympify()`
 
+UPDATE: The string fallback in `sympify` was deprecated in SymPy 1.6 and is now
+an error in SymPy 1.13 and later. Keeping this behaviour around even in
+deprecated form was too problematic so it is now removed entirely. In the cases
+where the string fallback would have been used a `SympifyError` will now be
+raised instead.
```

Comment:
The deprecation policy does not seem to specify what to do in this situation when removing the deprecated functionality. What is supposed to happen in the active deprecation docs?

https://docs.sympy.org/latest/contributing/deprecations.html

Reply:
I would suggest writing it in a declarative rather than defensive manner:

`sympify` would originally convert an unrecognized object to a string and retry sympification. This was deprecated in SymPy 1.6 and was removed in SymPy 1.13. To sympify such objects the user must explicitly pass the string version of the object.

Reply:
I've edited the existing text and moved it to the 1.13 deprecations section.

Reply:
For anyone wanting the fuller story on why that feature was discontinued, your history provides a very nice review.